CODEOWNERS

```diff
@@ -0,0 +1,143 @@
+lib/vdsm/metrics/                       @tinez
+lib/vdsm/mkimage.py                     @mz-pdm
+lib/vdsm/moduleloader.py                @tinez
+lib/vdsm/momIF.py                       @mz-pdm
+lib/vdsm/numa.py                        @mz-pdm
+lib/vdsm/osinfo.py                      @tinez
+lib/vdsm/ppc64HardwareInfo.py           @mz-pdm
+lib/vdsm/protocoldetector.py            @tinez
+lib/vdsm/rpc/                           @tinez
+lib/vdsm/schedule.py                    @nirs @mz-pdm
+lib/vdsm/sslutils.py                    @tinez
+lib/vdsm/supervdsm_server.py            @tinez
+lib/vdsm/taskset.py                     @nirs
+lib/vdsm/throttledlog.py                @mz-pdm
+lib/vdsm/tool/                          @tinez
+lib/vdsm/utils.py                       @tinez
+lib/vdsm/v2v.py                         @mz-pdm
+lib/vdsm/vdsm.conf.sample               @tinez
+lib/vdsm/vdsmd.py                       @tinez
+lib/vdsm/virtsparsify.py                @mz-pdm
+lib/vdsm/virtsysprep.py                 @mz-pdm
+
+# Profiling
+lib/vdsm/profiling/                     @nirs
+lib/vdsm/profiling/memory
+
+# Common
+lib/vdsm/common/                        @tinez
+lib/vdsm/common/api.py                  @tinez @nirs
+lib/vdsm/common/base26.py               @mz-pdm
+lib/vdsm/common/cache.py                @nirs @tinez
+lib/vdsm/common/cmdutils.py             @tinez @nirs
+lib/vdsm/common/commands.py             @tinez @nirs
+lib/vdsm/common/compat.py               @tinez @nirs
+lib/vdsm/common/concurrent.py           @nirs
+lib/vdsm/common/config.py.in            @tinez
+lib/vdsm/common/constants.py.in         @tinez
+lib/vdsm/common/conv.py                 @mz-pdm
+lib/vdsm/common/cpuarch.py              @mz-pdm
+lib/vdsm/common/define.py               @mz-pdm
+lib/vdsm/common/dsaversion.py.in        @tinez
+lib/vdsm/common/errors.py               @nirs
+lib/vdsm/common/eventfd.py              @tinez
+lib/vdsm/common/exception.py            @nirs @mz-pdm
+lib/vdsm/common/filecontrol.py          @tinez
+lib/vdsm/common/fileutils.py            @almusil @nirs @mz-pdm
+lib/vdsm/common/function.py             @almusil
+lib/vdsm/common/glob.py                 @nirs @vjuranek
+lib/vdsm/common/hooks.py                @tinez
+lib/vdsm/common/hostdev.py              @mz-pdm
+lib/vdsm/common/hostutils.py            @almusil
+lib/vdsm/common/libvirtconnection.py    @mz-pdm
+lib/vdsm/common/lockfile.py             @tinez @nirs
+lib/vdsm/common/logutils.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/nbdutils.py             @nirs
+lib/vdsm/common/osutils.py              @tinez
+lib/vdsm/common/panic.py                @nirs @tinez
+lib/vdsm/common/password.py             @nirs @mz-pdm @tinez
+lib/vdsm/common/pki.py                  @tinez
+lib/vdsm/common/proc.py                 @almusil
+lib/vdsm/common/properties.py           @nirs @vjuranek
+lib/vdsm/common/pthread.py              @mz-pdm @nirs @tinez
+lib/vdsm/common/response.py             @mz-pdm
```

Comment:
Doesn't this belong to infra?

Reply:
I think it is used only in virt code. In storage we have the dispatcher. I did not check network.

@almusil do you use the response.py module?

Reply:
It's also used in some common parts, such as jsonrpcvdscli.py or API.py, apparently not only in connection with virt.

Reply:
Yes it is used by many modules:

```
lib/vdsm/API.py:from vdsm.common import response
lib/vdsm/clientIF.py:from vdsm.common import response
lib/vdsm/common/api.py:from . import response
lib/vdsm/jobs.py:from vdsm.common import response
lib/vdsm/jsonrpcvdscli.py:from vdsm.common import response
lib/vdsm/v2v.py:from vdsm.common import response
lib/vdsm/virt/backup.py:from vdsm.common import response
lib/vdsm/virt/migration.py:from vdsm.common import response
lib/vdsm/virt/recovery.py:from vdsm.common import response
lib/vdsm/virt/secret.py:from vdsm.common import response
lib/vdsm/virt/vm.py:from vdsm.common import response
lib/vdsm/virt/vmdevices/lease.py:from vdsm.common import response
lib/vdsm/virt/vmpowerdown.py:from vdsm.common import response
tests/API_test.py:from vdsm.common import response
tests/api_response_test.py:from vdsm.common import response
tests/response_test.py:from vdsm.common import response
tests/virt/backup_test.py:from vdsm.common import response
tests/virt/device_test.py:from vdsm.common import response
tests/virt/diskreplicate_test.py:from vdsm.common import response
tests/virt/drive_extension_test.py:from vdsm.common import response
tests/virt/livemerge_test.py:from vdsm.common import response
tests/virt/powerdown_test.py:from vdsm.common import response
tests/virt/seal_job_test.py:from vdsm.common import response
tests/virt/v2v_test.py:from vdsm.common import response
tests/virt/vm_test.py:from vdsm.common import response
tests/virt/vmfakelib.py:from vdsm.common import response
tests/virt/vmlease_test.py:from vdsm.common import response
tests/virt/vmmigration_test.py:from vdsm.common import response
tests/virt/vmoperations_test.py:from vdsm.common import response
tests/virt/vmrecovery_test.py:from vdsm.common import response
tests/virt/vmsecret_test.py:from vdsm.common import response
```

So it looks like infa and @tinez can own it.